src/ui/MessageInput/index.tsx

```diff
@@ -170,6 +170,16 @@ const MessageInput = React.forwardRef<HTMLInputElement, MessageInputProps>((prop
     [],
   );
 
+  /**
+   * Whenever user given disabled is set to false, set focus.
+   */
+  useEffect(() => {
+    const textField = internalRef?.current;
+    if (textField && !disabled) {
+      textField.focus();
+    }
+  }, [disabled]);
```

Comment:
This caused customer issue.
https://sendbird.slack.com/archives/C06ML9924L9/p1717420929650799